themes/default/content/blog/pulumi-release-notes-m61/index.md

```diff
@@ -0,0 +1,428 @@
+                }
+            },
+            // By default Release resource will wait till all created resources
+            // are available. Set this to true to skip waiting on resources being
+            // available.
+            SkipAwait = false,
+        };
+
+        var release = new Release("redis-helm", releaseArgs);
+
+        // We can lookup resources once the release is installed. The release's
+        // status field is set once the installation completes, so this, combined
+        // with `skip_await=False` above, will wait to retrieve the Redis master
+        // ClusterIP till all resources in the Chart are available.
+        var status = release.Status;
+        var service = Service.Get("redist-master-svc", Output.All(status).Apply(
+            s => $"{s[0].Namespace}/{s[0].Name}-redis-master"));
+        this.RedisMasterClusterIP = service.Spec.Apply(spec => spec.ClusterIP);
+    }
+
+    [Output]
+    public Output<string> RedisMasterClusterIP { get; set; }
+}
+
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+Learn more in the [announcement blog post]({{< relref "/blog/full-access-to-helm-features-through-new-helm-release-resource-for-kubernetes" >}})
+
+### New resources in the Azure Native provider
+
+We shipped new versions of the Azure Native provider (1.22.0 through 1.28.0) that collectively added 15 new resources across services like SQL Server, Kusto, DocumentDB, and Power BI that you can now manage with the Azure Native provider.
+
+[See the full list](https://github.com/pulumi/pulumi-azure-native/blob/master/CHANGELOG.md#1280-2021-09-13)
+
+## Pulumi CLI and core technologies
+
+In this milestone, we shipped Pulumi versions [3.10.2](https://github.com/pulumi/pulumi/releases/tag/v3.10.2) through [3.12.0](https://github.com/pulumi/pulumi/releases/tag/v3.12.0). The full list of changes in each version is available [in the changelog](https://github.com/pulumi/pulumi/releases); read on to learn about some of the biggest changes.
+
+### New `pulumi about` command for easier troubleshooting
+
+To make it easier to troubleshoot errors in the Pulumi CLI, we've added a new `pulumi about` command that collects useful information like the plugins you have installed, the OS and architecture you're running, the backend that's storing your state, your CLI version, and more.
+
+![A screenshot of the output of the pulumi about command](pulumi-about.png)
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi/issues/2715)
+
+### `pulumi config set` hides secret values
+
+Previously, setting a config value using `pulumi config set` would write the value to `stdout`, which could be undesirable if the config value was a secret. Now, config values that look like secrets will be hidden and the config key name will be written to `stdout` instead.
+
+[Learn more in this GitHub PR](https://github.com/pulumi/pulumi/pull/7327)
+
+### Dynamic providers in Python can now given a custom name
```

Comment:
Should we highlight the improvements/fixes from external contributors? Maybe thanking them by name or GH username, and linking to their GH profile?

Reply:
Fantastic idea, thanks! Added.